### PR TITLE
add test case for user list

### PR DESCRIPTION
**Pull request by mkudlej**

### CODE REVIEW

**Review by ltrilety — APPROVED** [review]

File: web/tendrl/user_list.rst [hunk 1]
```diff
@@ -0,0 +1,46 @@
+User list
+**********
+
+:authors: 
+          - mkudlej@redhat.com
+
+Description
+===========
+
+Check user list testcase and compare it with `https://redhat.invisionapp.com/share/8QCOEVEY9#/screens/244738633`.
```
Comment (mbukatov) [inline web/tendrl/user_list.rst]: Remove ` characters for the url to become a proper link.

**Review by mbukatov — CHANGES_REQUESTED** [review]
Wrong link syntax, otherwise looks good.

File: web/tendrl/user_list.rst [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        * User ID
+        * Last name
+        * Role
+        * Status
+   :result:
+      Users can be found by these 4 attributes.
+
+Teardown
+========
+#. Log out
```
Comment (fbalak) [inline web/tendrl/user_list.rst]: I would use same formatting as in `Test Steps` part.

**Review by fbalak — APPROVED** [review]
LGTM

**Pull request merged**